src/sentry/static/sentry/app/utils/discover/fieldRenderers.tsx

```diff
@@ -296,13 +295,19 @@ const SPECIAL_FUNCTIONS: SpecialFunctions = {
 
       const palette = new Array(10).fill(theme.purpleDarkest);
       const score = Math.floor((userMisery / Math.max(uniqueUsers, 1)) * palette.length);
-      const miseryLimit = parseInt(userMiseryField.split('_').pop(), 10);
-      const title = `${userMisery} out of ${uniqueUsers} unique users waited more than ${4 *
-        miseryLimit}ms`;
+      const miseryLimit = parseInt(userMiseryField.split('_').pop() || '', 10);
+      const title = t(
+        '%s out of %s unique users waited more than %sms',
+        userMisery,
+        uniqueUsers,
+        4 * miseryLimit
+      );
```

Comment:
We need to use `tct` here if string formatting takes more than 1 argument, otherwise, it's impossible for translators to be able to change the order of strings (since sentence structures can differ between languages).